.kokoro/docker/Dockerfile

```diff
@@ -103,7 +103,7 @@ RUN set -ex \
     && export GNUPGHOME="$(mktemp -d)" \
     && echo "disable-ipv6" >> "${GNUPGHOME}/dirmngr.conf" \
     && /tmp/fetch_gpg_keys.sh \
-    && for PYTHON_VERSION in 2.7.18 3.6.10 3.7.7 3.8.3; do \
+    && for PYTHON_VERSION in 2.7.18 3.6.10 3.7.7 3.8.3 3.9.0; do \
```

Comment:
I don't see the corresponding update to gpg keys? Is it the same as the 3.8 key? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.kokoro/docker/fetch_gpg_keys.sh

Reply:
No idea - any hints on finding this out?

Reply:
3.9 uses the same key as 3.8.  https://github.com/googleapis/testing-infra-docker/blob/307a7233dbb3e5ac5c65d364de16a4770233549b/python/googleapis/python-multi/Dockerfile#L89-L90

Reply:
👍 The comment in the linked file should be updated to include that (comment currently says 3.8 and it might be confusing. While you are at it you can add 3.10 and 3.11 since that is around the corner.

```
RUN set -ex \
  && gpg --keyserver ha.pool.sks-keyservers.net --recv-keys \
      # 2.7.17 (Benjamin Peterson)
      C01E1CAD5EA2C4F0B8E3571504C367C218ADD4FF \
      # 3.4.10, 3.5.10 (Larry Hastings)
      97FC712E4C024BBEA48A61ED3A5CA953F73C700D \
      # 3.6.12, 3.7.9 (Ned Deily)
      0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D \
      # 3.8.6 and 3.9.0 (Łukasz Langa)
      E3FF2839C048B25C084DEBE9B26995E310250568 \
      # 3.10.x and 3.11.x (Pablo Galindo Salgado)
      A035C8C19219BA821ECEA86B64E628F8D684696D
```

Reply:
Done for `fetch_gpg_keys.sh`